ssg/build_yaml.py

```diff
@@ -1004,6 +1006,12 @@ def to_xml_element(self):
         add_sub_element(rule, 'description', self.description)
         add_sub_element(rule, 'rationale', self.rationale)
 
+        if self.metadata:
+            meta = add_sub_element(rule, 'metadata')
+            for meta_tag, meta_text in self.metadata.items():
+                meta_tag_with_ns = "xml:{tag}".format(tag=meta_tag)
```

Comment:
Is there a way to not hard-code the namespace here? Will need dublin-core elements here as well.